src/main/java/org/kohsuke/github/GHRepositorySearchBuilder.java

```diff
@@ -52,6 +54,40 @@ public GHRepositorySearchBuilder forks(String v) {
         return q("forks:" + v);
     }
 
+    /**
+     * Searching in forks
+     *
+     * @param fork
+     *            search mode for forks
+     *
+     * @return the gh repository search builder
+     *
+     * @see <a href=
+     *      "https://docs.github.com/en/github/searching-for-information-on-github/searching-on-github/searching-in-forks">Searching
+     *      in forks</a>
+     *
```

Comment:
Hm, so the default (if this value isn't supplied is to not search forks, right?  Should that be a value in the enum?  `Forks.DEFAULT` which removes that filter if present?  

Also, you probable want to be explicit about the limitations, both here and in the JavaDoc for the enum: 

> To show forks with more stars than the parent repository in code search results, add `fork:true` or `fork:only` to your query.

Reply:
@bitwiseman yep, according to docs we ignore forks by default.
Do we have a use case when we need Forks.DEFAULT?

Reply:
Someone might want to provide a default builder that sets `fork:true` and then change it to default in only a few places in their code.  I see you've made the change.  Thanks!